lib/validate.js

```diff
@@ -59,7 +59,7 @@ module.exports = function( message, options ) {
 				" allowed. Was: " + line.substring( 0, 20 ) + "[...]" );
 		}
 		// Ticket references
-		if ( /^(clos|fix|resolv)(e[sd]|ing)/i.test( line ) ) {
+		if ( /^(clos|fix|resolv)(e[sd]|ing)/i.test( line ) && index > 0 ) {
```

Comment:
Any reason to have the index check last? Not that its important, but would "faster" to check that first.